postcss-selector-parser.d.ts

```diff
@@ -311,6 +351,51 @@ declare namespace parser {
          * The attribute name after having been qualified with a namespace.
          */
         readonly qualifiedAttribute: string;
+
+        /**
+         * The case insensitivity flag or an empty string depending on whether this
+         * attribute is case insensitive.
+         */
+        readonly insensitiveFlag : 'i' | '';
+
+        /**
+         * Returns the attribute's value quoted such that it would be legal to use
+         * in the value of a css file. The original value's quotation setting
+         * used for stringification is left unchanged. See `setValue(value, options)`
+         * if you want to control the quote settings of a new value for the attribute or
+         * `set quoteMark(mark)` if you want to change the quote settings of the current
+         * value.
+         *
+         * You can also change the quotation used for the current value by setting quoteMark.
+         **/
+        quoteValue(options?: SmartQuoteMarkOptions): string;
```

Comment:
I would like to see parity with `setValue` and have a `getQuoteValue`. This method reads to me like you're modifying the Attribute to add quotes, not getting the quoted value.

Reply:
👍